static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx

```diff
@@ -203,9 +205,11 @@ const ProductSummary = () => {
           </Col>
           {product?.canBeTrialled && productUser !== ProductUsers.myself ? (
             <Col size={12}>
-              <StatusLabel appearance="positive">
-                Free trial available
-              </StatusLabel>
+              <a href="/engage/Ubuntu-Pro-1-month-trial">
+                <StatusLabel appearance="positive">
+                  Free trial available
+                </StatusLabel>
+              </a>
```

Comment:
It's working fine, but I noticed it doesn't look clickable, so users might not realize it is.  Should we move the `a` tag inside? What do you think?
```suggestion
          <StatusLabel appearance="positive">
              <a href="/engage/Ubuntu-Pro-1-month-trial">
                  Free trial available
              </a>
            </StatusLabel>
```

Reply:
how does this look?

![image](https://github.com/user-attachments/assets/383f6401-d4ff-4b3b-956c-7fa40eb271bb)

Reply:
Yep, this looks better for me